agent/config/runtime_test.go

```diff
@@ -4076,7 +4078,7 @@ func TestFullConfig(t *testing.T) {
 			"key_file": "IEkkwgIA",
 			"leave_on_terminate": true,
 			"limits": {
-				"http_max_conns_per_client": 250,
+				"http_max_conns_per_client": 100,
```

Comment:
Had to change it because it was too high for my mac on default settings for file handles.